components/Mediator.vue

```diff
@@ -528,6 +528,12 @@ export default {
       try {
         response = await navigator.credentialMediator.ui.selectCredentialHint(
           event.hint.hintOption);
+        if(!response) {
+          this.selectedHint = null;
+          this.rememberChoice = true;
+          // clear site choice
+          setSiteChoice({relyingOrigin, credentialHandler: null});
+        }
```

Comment:
Setting `this.selectedHint = null` and `this.rememberChoice = true` are redundant because they will be set to those values when `reset()` is called -- which executes below when `canceled` is false which it will be in this case. So, I think the only thing that needs to happen that's different is clearing any existing site choice when the response is `null`.

```suggestion
        if(!response) {
          // clear site choice
          setSiteChoice({relyingOrigin, credentialHandler: null});
        }
```

Reply:
Removed here fda3ee6